aioredis/sentinel.py

```diff
@@ -175,10 +175,15 @@ def create_sentinel(sentinels, *, db=None, password=None,
 
     sentinels_connections = []
     for hostname, port in sentinels:
-        sentinel = yield from create_redis((hostname, port), db=db,
-                                           password=password,
-                                           encoding=encoding, loop=loop)
-        sentinels_connections.append(sentinel)
+        try:
+            sentinel = yield from create_redis((hostname, port), db=db,
+                                               password=password,
+                                               encoding=encoding, loop=loop)
+            sentinels_connections.append(sentinel)
+        except OSError:
+            pass
```

Comment:
I think we should add `warnings.warn` or add logging, it is good to know that one of sentinels is dead.